docs/details/victoria-metrics.md

```diff
@@ -33,6 +33,28 @@ As a result of the move to VictoriaMetrics some direct Prometheus paths are no l
 | `/prometheus/status`            | Some information at `/prometheus/metrics`. High cardinality metrics information at `/prometheus/api/v1/status/tsdb`.
 | `/prometheus/targets`           | `/victoriametrics/targets`
 
+
+
+## Environment variables
+
+
+PMM sets the following flags that allow users to set all other [VictoriaMetrics parameters](https://docs.victoriametrics.com/#list-of-command-line-flags) as environment variables:
+
+- `-envflag.enable`  
+- `-envflag.prefix=VM_` - The environment variable must be prepended with `VM_`.
+
```

Comment:
not sure why we need this at all,  as this is a part of "how we internally  run VM."
(this might be  in Architecture, but  not in this section ) 

Only example -e VM_..... make sense for PMM users.